### DISCUSSION TITLE
"Cannot read the array length because revisions is null"

### DISCUSSION THREAD

**Discussion by zenon26** [freeplane/freeplane#854] [Help]
impossible ouvrir des cartes car ce message apparait... (version 10.5)
![Freeplane_erreur](https://user-images.githubusercontent.com/119207725/204143720-e325ab42-266a-4a61-bf21-95caf5f79680.png)

**Comment by euu2021** [comment]
Does that work: create a new map, save it, close Freeplane, and then open this map?
